portfolio/src/main/webapp/index.html

```diff
@@ -62,7 +63,12 @@ <h1>Zoe Mitchell</h1>
             <div id="top"> 
```

Comment:
Where is the closing `</div>` for this block?

Reply:
Could you fix the indentation for the elements within this div? It's not clear where the div starts and ends